handlers/eth1Account.go

```diff
@@ -166,6 +169,15 @@ func Eth1Address(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
+
+	code, err := eth1data.GetCodeAt(ctx, common.BytesToAddress(addressBytes))
+	if err != nil {
+		logger.WithError(err).Errorf("error retrieving code data for address %v", address)
+	}
+	isContract := len(code) != 0
```

Comment:
This is adding duplicated code, see here
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/1f2df8d694574a6f673a4d309f5499c973cc560b/eth1data/eth1data.go#L80-L84

Please add a `IsContract` function (most likely in `eth1data.go´?) and use that.

Reply:
Done.

Reply:
> I ping @recy21 though, maybe he has a different opinion.

No, lets do like that, thanks!